test/docTest.js.html

```diff
@@ -313,18 +315,41 @@
             assert.equal( element.hasClass('added-class-for-test'), true);
           });
 
+          it('should add class to the given element with multiple classes', function(){
+            var element = $('#add-remove-class-test').addClass('class-1 class-2');
+            assert.equal( element.size, 1);
+            assert.equal( element.hasClass('class-1'), true);
+            assert.equal( element.hasClass('class-2'), true);
+          });
+
           it('should remove class to the given element', function(){
             var element = $('#add-remove-class-test').removeClass('to-remove-class-for-test');
             assert.equal( element.hasClass('to-remove-class-for-test'), false);
           });
 
+          it('should remove class to the given element with multiple classes', function(){
+            var element = $('#add-remove-class-test').addClass('class-1 class-2');
```

Comment:
What about adding a third class just to guarantee that it still remains after removing the other two?